languages/elixir/exercises/concept/booleans/.docs/after.md

```diff
@@ -1,31 +1,31 @@
 Elixir represents true and false values with the boolean type. There are only two values -- _true_ and _false_. These values can be bound to a variable and combined with boolean operators (`and/2`, `or/2`, `not/1`):
 
 ```elixir
-is_true = true and true
-is_false = true and false
+true_variable = true and true
+false_variable = true and false
 
-is_true = false or true
-is_false = false or false
+true_variable = false or true
+false_variable = false or false
 
-is_true = not false
-is_false = not true
+true_variable = not false
+false_variable = not true
 ```
 
-Boolean operators use _short-circuit evaluation_, which means that right-hand side of the operator is only evaluated with needed.
+Boolean operators use _short-circuit evaluation_, which means that expression on the right-hand side of the operator is only evaluated if needed.
 
-Each of the operators has a different precendence, where `not/1` is evaluated before `and/2`, `or/2`.
+Each of the operators has a different precedence, where `not/1` is evaluated before `and/2` and `or/2`.
```

Comment:
Would it be worth mentioning in text that parentheses can be used to force the evaluation order? As demonstrated in the example code.